src/components/table/makeSelectable.js

```diff
@@ -348,22 +370,30 @@ function makeSelectable(BaseTable) {
         };
 
         render() {
-            const { className, data } = this.props;
+            const { className, data, enableBlurDetection } = this.props;
             const { focusedIndex } = this.state;
             const focusedItem = data[focusedIndex];
 
-            return (
-                <Hotkeys configs={this.getHotkeyConfigs()}>
-                    <BaseTable
-                        {...this.props}
-                        className={classNames(className, 'is-selectable')}
-                        focusedIndex={focusedIndex}
-                        focusedItem={focusedItem}
-                        onRowClick={this.handleRowClick}
-                        onRowFocus={this.handleRowFocus}
-                    />
-                </Hotkeys>
+            let table = (
+                <BaseTable
+                    {...this.props}
+                    className={classNames(className, 'is-selectable')}
+                    focusedIndex={focusedIndex}
+                    focusedItem={focusedItem}
+                    onRowClick={this.handleRowClick}
+                    onRowFocus={this.handleRowFocus}
+                />
             );
+
+            if (enableBlurDetection) {
+                table = (
+                    <div onBlur={this.handleTableBlur} onFocus={this.handleTableFocus}>
```

Comment:
Can we pass these to the `BaseTable` component directly? otherwise, we introduce another wrapping `div`.

Reply:
Great feedback! This actually simplifies the BUIE change quite a bit (but puts the burden on the consumers). The change is now automatically backward compatible, so there is no need for a boolean prop anymore.